Model/CodedCategoryRepository.php

```diff
@@ -116,6 +116,8 @@ public function deleteByIdentifier($categoryCode)
         } finally {
             $this->releaseLock($categoryCode);
         }
+
+        return true;
```

Comment:
Shouldn't we be removing the `bool` return and keeping the `void` in this situation? The functionality has never returned anything before

Reply:
hm, in the interface the signature contains the bool return statement

Reply:
Ok, that's fine in that case 👍